src/Sarif.Viewer.VisualStudio/FixSuggestedActionsSource.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft. All rights reserved. 
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.Language.Intellisense;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+
+namespace Microsoft.Sarif.Viewer
+{
+    internal class FixSuggestedActionsSource : ISuggestedActionsSource
```

Comment:
>ISuggestedActionsSource [](start = 47, length = 23)

Does this actually derive from IDisposable. I'm assuming the fixes will be disposable due to persistent spans being disposable

Reply:
Yes, it does.

---
In reply to: [509475985](https://github.com/microsoft/sarif-visualstudio-extension/pull/228#discussion_r509475985) [](ancestors = 509475985)